swagger.yaml

```diff
@@ -3249,6 +3290,54 @@ definitions:
       worker:
         $ref: '#/definitions/host-requirements-role'
 
+  preflight-requirements:
+    type: object
+    properties:
+      operators:
+        description: Preflight operators requirements
+        type: array
+        items:
+          $ref: '#/definitions/operator-requirements'
+      ocp:
+        description: Preflight OCP requirements
+        $ref: '#/definitions/host-type-requirements-wrapper'
+
+  operator-requirements:
+    type: object
+    properties:
+      operator_name:
+        description: Name of the operator
+        type: string
+      dependencies:
+        description: List of other operators that are required to be installed
+        type: array
+        items:
+          type: string
```

Comment:
What's the exact format of this string? Is it the operator name or some ID/URL? maybe add an example / more detailed description of how operators are identified